pyServer/AzureDiskInspectService.py

```diff
@@ -472,6 +472,16 @@ def do_POST(self):
                     self.telemetryLogger.info('WARNING: Received timeout override is invalid: {0}. Default will be used.'.format(timeoutInMinsStr))
 
             timeoutInMins = int(timeoutInMinsStr)
+
+            self.telemetryLogger.info('Received timeout value: ' + str(timeoutInMins)+ ' min(s)')
+
+            if timeoutInMins < 20:
```

Comment:
Would you make these constants so that can be named and more easily located should they need to be tuned?  (here and below)

Reply:
Will do!

Reply:
First one is still here  :)  perhaps:
`timeoutInMins <= DEFAULT_TIMEOUT_IN_MINS`  ?

Reply:
Done :)